packages/mesa/build.sh

```diff
@@ -15,21 +16,33 @@ TERMUX_PKG_REPLACES="libmesa"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 --cmake-prefix-path $TERMUX_PREFIX
 -Dcpp_rtti=false
--Dgbm=enabled
+-Dgbm=disabled
 -Dopengl=true
 -Degl=enabled
 -Degl-native-platform=x11
--Dgles1=disabled
+-Dgles1=enabled
```

Comment:
Do not enable GLES1. It will (possibly) make `libglvnd` unable to work.

Reply:
Need to check